support/ppPrint.h

```diff
@@ -0,0 +1,49 @@
+#pragma once
+
+#ifdef SPDLOG_ENABLED
```

Comment:
Any macros should be project name prefixed. So, this should become `PUMIPIC_SPDLOG_ENABLED`